code/datums/components/riding.dm

```diff
@@ -316,12 +316,17 @@
 /datum/component/riding/cyborg/force_dismount(mob/living/M)
 	var/atom/movable/AM = parent
 	AM.unbuckle_mob(M)
+	if(AM.throwcooldown)
```

Comment:
```suggestion
	var/mob/living/silicon/robot/S = AM
	if(S?.throwcooldown)
```
currently can't read variables from robot since the variable being used isn't that specific, this should probably work if it's switched out for all the bits where AM.throwcooldown is mentioned